### ISSUE TITLE
hasktags --ignore-close-implementation seems broken

### ISSUE THREAD

**Issue by ggVGc** [aloiscochard/codex#52]
Setting `tagsCmd` to `hasktags --ignore-close-implementation` seems to only have effect on the files in the current project, and not in any of the dependencies. This is a bit annoying since without it the tags file ends up having duplicates of every function, once for the declaration and once for implementations.

**Comment by wolftune** [comment]
+1

**Comment by aloiscochard** [comment]
@ggVGc I do not see how the behavior you describe could happen, besides the fact that you are getting outdated tags file from the cache.

Could you try to run `codex cache clean` **before** re-running `codex update`, this required if you change the tagger configuration and you had used codex before.

@wolftune you mean in the default configuration, similar to how `follow-symlinks` is defined? (https://github.com/aloiscochard/codex/blob/master/src/Codex.hs#L71-L73)

**Comment by wolftune** [comment]
@aloiscochard yes, that *sounds* like exactly what I mean. I didn't investigate thoroughly, but yes, --ignore-close-implementation should be always on as default for hasktags. The truth is, it should be made the default behavior for hasktags itself, and then it wouldn't be a codex issue, but for now…
